wcag/3.3.8.md

```diff
@@ -2,4 +2,27 @@
 
 (Level AA)
 
-Placeholder
+This applies directly as written, and as described in [Intent from Understanding Success Criterion 3.3.8](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum.html), “the Web site” with “a Web site, screen, or application”.
```

Comment:
Missing `replacing` before "the Web site".

Maybe just substitute `Web site` with `Application`? As application includes all screens, views, components, etc.